packages/block-library/src/heading/style.native.scss

```diff
@@ -0,0 +1,5 @@
+@import "variables.scss";
```

Comment:
We'll have to remove this, this was dropped as part of the merge PR (we auto import `_variables.scss` now)